### ISSUE TITLE
508-defect-2 [Headings]: Search results for resources and support should be consistent with other search heading orders

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#29743]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a screen reader user, I expect search results on resources and support to be consistent with other search pages on VA.gov.

## Details

On VA.gov and Find a Form's search pages, "Showing 1 - 3 of 3 results..." is usually the `h2` which indicates that this is where users can find _search results_ as opposed to the `h1` which usually contains the search itself. Screen reader users who have become familiar with VA.gov or Find a Form's heading orders may attempt to use the same shortcuts on resources and support only to fail e.g. (Using "3" as a shortcut to jump to the heading level 3s on the page which are search results on other pages).

## Acceptance Criteria

- [ ] [Ticket 29733 has been closed](https://github.com/department-of-veterans-affairs/va.gov-team/issues/29733)
- [ ] "Showing 1 - 3 of 3 results..." is a `h2`
- [ ] Search result links are `h3`s, not `h2`s

## Steps to Recreate

1. Enter `https://www.va.gov/resources/search/?query=veteran` in browser
2. Confirm that "Showing 1 of 3..." is not a heading

## Proposed Solution

To be consistent with VA.gov and Find a Form, I recommend:
- Changing "Showing 1-3 of 3...." to a `h2` and
- Updating search result links to `h3`s instead of `h2`s.

Those changes might look like this:
<img width="1326" alt="Screen Shot 2021-09-08 at 12 33 07 PM" src="https://user-images.githubusercontent.com/14154792/132549137-25b66c2f-62bc-4582-856a-7f62851bb581.png">

**Comment by srsuddath** [comment]
This ticket has been accomplished as part of the following PR
https://github.com/department-of-veterans-affairs/vets-website/pull/19400
